.github/workflows/phpunit-mysql.yml

```diff
@@ -100,7 +101,7 @@ jobs:
           DB_PORT: 4444
         run: |
           mkdir data
-          ./occ maintenance:install --verbose --database=mysql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass password
+          ./occ maintenance:install --verbose --database=mysql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass admin
```

Comment:
ah? I see some repo with admin, the other with password

Reply:
In the oci workflow it's `admin`

Reply:
Seems like it doesn't change anything according to the tests passing

Reply:
yeah admin admin is new fixed default.
Talk is running integration tests not from this action, so no result change